### PR TITLE
ci: run all checks on pull request

### PR DESCRIPTION
**Pull request by simonseyock**
## Description

run all checks on every pull request.

Also simplifies the reexports in the index.ts

## Pull request type

- [ ] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the
  [BSD-2-Clause](https://github.com/terrestris/react-geo/blob/main/LICENSE).
- [ ] I have followed the [guidelines for contributing](https://github.com/terrestris/react-geo/blob/main/CONTRIBUTING.md).
- [ ] The proposed change fits to the content of the [Code of Conduct](https://github.com/terrestris/react-geo/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally).
- [ ] I have added a screenshot/screencast to illustrate the visual output of my update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +60,6 @@
     "lint:styleguide": "eslint -c .eslintrc.styleguide.cjs src/",
     "lint:styleguide-fix": "eslint -c .eslintrc.styleguide.cjs src/ --fix",
     "prepare": "husky",
-    "pretest": "npm run typecheck && npm run lint",
```
Comment (dnlkoch) [inline package.json:62]: Can we remove the `npm run test -- --coverage` in the `build` stage then as well?

**Review by dnlkoch — APPROVED** [review]
Thanks!

File: package.json (hunk 1 above)
Comment (simonseyock) [reply package.json:62]: Ok, it seems that my change was actually not needed. But this way it is clearer what happens. I remove the test run form the build script.

**Pull request merged**